src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -32,20 +33,20 @@ export interface UserProfilePictureProp extends ContainerClassName {
 export const UserProfilePicture: React.FC<UserProfilePictureProp> = ({
   user,
   isAudioEffectDisabled = true,
-  containerClassName,
   reactionPosition = "right",
   showStatus = false,
   isVideoEnabled = true,
   size,
+  containerClassName,
```

Comment:
[Q] Is this still used?

Reply:
yes, in the `src/hooks/auditorium/useAuditoriumGrid.tsx`